recipes/sdl_ttf/all/conanfile.py

```diff
@@ -82,22 +98,28 @@ def build(self):
 
     def package(self):
         copy(self, "COPYING.txt", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "LICENSE.txt", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
         cmake = CMake(self)
         cmake.install()
         rmdir(self, os.path.join(self.package_folder, "cmake"))
         rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
         rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
         rmdir(self, os.path.join(self.package_folder, "SDL2_ttf.framework"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
 
     def package_info(self):
+        suffix = "-static" if Version(self.version) >= "2.20.0" and not self.options.shared else ""
+
         self.cpp_info.set_property("cmake_file_name", "SDL2_ttf")
-        self.cpp_info.set_property("cmake_target_name", "SDL2_ttf::SDL2_ttf")
+        self.cpp_info.set_property("cmake_target_name",f"SDL2_ttf::SDL2_ttf{suffix}")
         self.cpp_info.set_property("pkg_config_name", "SDL2_ttf")
 
         self.cpp_info.includedirs.append(os.path.join("include", "SDL2"))
-        self.cpp_info.libs = ["SDL2_ttf"]
-        self.cpp_info.requires = ["freetype::freetype", "sdl::libsdl2"]
+        self.cpp_info.libs = [f"SDL2_ttf{'-static' if is_msvc(self) and not self.options.shared else ''}{'d' if self.settings.build_type == 'Debug' else ''}"]
```

Comment:
The `d` suffix is only with the latest releases.
But it's probably easier to just disable the suffix.
It won't be used anymore for the 3 series

Reply:
@madebr 
I see. Let me disable it.
Thanks!

Reply:
@madebr 
Would you answer my question?
When I build sdl_ttf/2.20.1 on MSVC, the library name is `SDL2_ttf.lib`.
There is no `SDL2_ttf-static` library.

```
sdl_ttf/2.20.1@toge/stable package(): Packaged 1 '.h' file: SDL_ttf.h
sdl_ttf/2.20.1@toge/stable package(): Packaged 1 '.lib' file: SDL2_ttf.lib
sdl_ttf/2.20.1@toge/stable package(): Packaged 1 '.txt' file: LICENSE.txt
sdl_ttf/2.20.1@toge/stable: Package '32fe13d8502f0ebb810dd173f5c88c1a29da72d9' created
sdl_ttf/2.20.1@toge/stable: Created package revision 8198428ae16d89808f97ee9fafbc4d78
```

May SDL_ttf-static is the name of the library in version 3.0?

Reply:
Oh, you're right.
It's in git but not yet released.
It will only be present in the next minor sdl_ttf release
https://github.com/libsdl-org/SDL_ttf/blob/460d79bee36a6edacbc4c072b8409ec7e650b214/CMakeLists.txt#L151
I apologize 1000 times!

Reply:
@madebr 
No problem.
Thank you for your review.
Your review helps me a lot!